docs/develop/built-in-ui-ids.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Find the IDs of built-in Office ribbon tabs
+description: Find the IDs of the built-in Office ribbon tabs.
+ms.date: 07/31/2024
+ms.topic: reference
+ms.localizationpriority: medium
+---
+
+<!-- 
+  This article is deliberately left out of the Office Add-ins TOC because 
+  it will be moving over to the M365 doc set as soon as that is up and running. 
+-->
+
+# Find the IDs of built-in Office ribbon tabs
+
+The following are valid built-in Office ribbon tab `id` values by application. Values in **bold** are supported in both desktop and online; and, for Outlook, in New Outlook. Others are supported only in desktop. For example, **TabHome** is supported in Word 2016 or later on Windows and Word on the web, but TabBlogPost is supported only in Word desktop.
```

Comment:
Not all screen readers may identify bold text. For accessibility, consider creating a table that lists which values are supported in desktop, online, or both.